exercises/alphametics/alphametics_test.rb

```diff
@@ -3,63 +3,72 @@
 require 'minitest/autorun'
 require_relative 'alphametics'
 
-# Test data version:
-# 8d8589f
+# Test data version: c1cb73f
 class AlphameticsTest < Minitest::Test
-  def test_solve_short_puzzle
+
+  def test_puzzle_with_three_letters
     # skip
-    expect = {
-      'I' => 1, 'B' => 9, 'L' => 0
-    }
-    actual = Alphametics.new.solve('I + BB == ILL')
-    assert_equal(expect, actual)
+    input = 'I + BB == ILL'
+    expected = { 'B' => 9, 'I' => 1, 'L' => 0 }
+    assert_equal expected, Alphametics.solve(input)
```

Comment:
Was: `Alphametics.new.solve(input)`
Now: `Alphametics.solve(input)`
Instantiating an object is unnecessary, so it now uses an instance method.